docs/src/booklet/6-solvers.md

```diff
@@ -0,0 +1,21 @@
+# QUBO Solvers
```

Comment:
MQLib, all ones, all zeros, and Qiskit missing, right?

Reply:
Right. all-ones/zeros are provided by configuring the `IdentitySampler`.

This section would better fit in Anneal.jl or an eventual QUBO.jl package docs.